net.werwolv.ImHex.yaml

```diff
@@ -0,0 +1,62 @@
+app-id: net.werwolv.ImHex
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: imhex
+rename-desktop-file: imhex.desktop
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --filesystem=home
+  - --device=dri
+
+modules:
+  - name: glfw
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DBUILD_SHARED_LIBS=ON
+      - -DGLFW_BUILD_EXAMPLES=OFF
+      - -DGLFW_BUILD_TESTS=OFF
+      - -DGLFW_BUILD_DOCS=OFF
+    sources:
+      - type: archive
+        url: https://github.com/glfw/glfw/releases/download/3.3.2/glfw-3.3.2.zip
+        sha256: 08a33a512f29d7dbf78eab39bd7858576adcc95228c9efe8e4bc5f0f3261efc7
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+
+  - name: mbedtls
```

Comment:
There are a bunch of unrelated binaries in `/app/bin` that probably come from this. They should be cleaned up.